efficient_full_table_scan_example_code/README.md

```diff
@@ -89,3 +89,54 @@ Done!
 
 Total Execution Time: 15m32s
 ```
+
+
+
+Generic GO installation and script setup into ~/gocqlcount
+====================================================
+
+E.g. for go 1.20.4:
+
+```
+cd ~
+
+GOVERSION=go1.20.4
+curl -o $GOVERSION.linux-amd64.tar.gz https://dl.google.com/go/$GOVERSION.linux-amd64.tar.gz
+
+tar xzvf $GOVERSION.linux-amd64.tar.gz
+
+ln -s $HOME/go $HOME/$GOVERSION 
+
+cat <<EOF > go_profile.sh
+#!/bin/bash
+export GOVERSION=$GOVERSION
+export GOROOT=\$HOME/\$GOVERSION
+
+export GOPATH=\$HOME/go
+
+export PATH=\$PATH:\$GOROOT/bin
+EOF
+
+chmod 755 go_profile.sh
+. go_profile.sh
+
+mkdir gocqlcount
+
+go mod init gocqlcount
+
+cd gocqlcount
+go get github.com/gocql/gocql
+go get github.com/alecthomas/kingpin/v2
```

Comment:
if you curl first, then you can run `go mod tidy` to download all necessary dependencies without need to manually get every each of them

Reply:
this didn't work well for me

and it points out that github.com/alecthomas/kingpin/v2 code is a bit dead

Reply:
resp. it's not dead, but doesn't really play well with go mod tidy ...

Reply:
both `go mod tidy` and `go get` should do essentially the same. What's the issue with `go mod tidy`?

Reply:
Hey @zimnx and @tarzanek!

Should we close this PR or you folks still want to finish it?

Reply:
fixed, problem was with directory where module was created